toolkit/data-sources/mock/src/lib.rs

```diff
@@ -1,22 +1,78 @@
-#[allow(unused_imports)]
-use std::sync::Arc;
+//! # Partner Chains Mocked Data Sources
+//!
+//! This crate provides mocked implementations of all data source interfaces used
+//! by the Partner Chain toolkit's components, that can be wired into a Substrate
+//! node to avoid having access to a real data source like the Db-Sync data sources
+//! provided by `partner_chain_db_sync_data_sources` crate.
+//!
+//! # Usage
+//!
+//! The mock data sources defined in this crate are meant to make it possible to run
+//! a Partner Chain node with as little additional work as possible. Because of that,
```

Comment:
Considering experience, I would strongly point that mock should not be used beyond very early stages of exploration and development and is not a good fit for public testnets, because it rules our interaction with Cardano.